### ISSUE TITLE
How to use this plugin?

### ISSUE THREAD

**Issue by nonunknown** [Leleat/AGodotCodeSnippetPlugin#1]
I've installed it on my godot and enabled it, but at README.MD there is no how to use, cuz I cant see the panel as demonstrated on preview.

**Comment by Leleat** [comment]
> I've installed it on my godot and enabled it, but at README.MD there is no how to use, cuz I cant see the panel as demonstrated on preview.

You open the popup with Control+Tab (it's mentioned in the first paragraph of the readme; I've bolded it now for easier readibility ;)).

PS.

if you are coming from the AssetLib, I recommend you download the addons/ folder manually and add it to your project. I added new features to make the plugin more comparable to the snippets of for example VSCode (placeholders, dropdown choice and tabstop jumping in numerically ascending order).

I still need to test it a bit more before I update the version in the AssetLib (like a week or so). It should be good to go though.
